{"uid":"97c51fbc1e4097b6","name":"test_login_logout[data0-chrome]","fullName":"tests.test_login#test_login_logout","historyId":"08c892752312637e60f78927c03310d7","time":{"start":1761124451641,"stop":1761124486922,"duration":35281},"status":"failed","statusMessage":"AssertionError: Login failed for user VaishnaviK123\nassert False","statusTrace":"browser_name = 'chrome', data = {'password': 'Vaishu@123', 'username': 'VaishnaviK123'}\n\n    @pytest.mark.parametrize(\"browser_name\", [\"chrome\", \"edge\"])\n    @pytest.mark.parametrize(\"data\", login_data)\n    def test_login_logout(browser_name, data):\n        # Get driver for the current browser\n        driver = DriverFactory.get_driver(browser_name=browser_name, headless=False)\n    \n        login_page = DemoBlazeLogin(driver)\n        login_page.open_url(\"https://www.demoblaze.com/\")\n    \n        success = login_page.login(data['username'], data['password'])\n>       assert success, f\"Login failed for user {data['username']}\"\nE       AssertionError: Login failed for user VaishnaviK123\nE       assert False\n\ntests\\test_login.py:53: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"AssertionError: Login failed for user VaishnaviK123\nassert False","statusTrace":"browser_name = 'chrome', data = {'password': 'Vaishu@123', 'username': 'VaishnaviK123'}\n\n    @pytest.mark.parametrize(\"browser_name\", [\"chrome\", \"edge\"])\n    @pytest.mark.parametrize(\"data\", login_data)\n    def test_login_logout(browser_name, data):\n        # Get driver for the current browser\n        driver = DriverFactory.get_driver(browser_name=browser_name, headless=False)\n    \n        login_page = DemoBlazeLogin(driver)\n        login_page.open_url(\"https://www.demoblaze.com/\")\n    \n        success = login_page.login(data['username'], data['password'])\n>       assert success, f\"Login failed for user {data['username']}\"\nE       AssertionError: Login failed for user VaishnaviK123\nE       assert False\n\ntests\\test_login.py:53: AssertionError","steps":[],"attachments":[{"uid":"e9e5df7e26df146","name":"stdout","source":"e9e5df7e26df146.txt","type":"text/plain","size":90}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_login"},{"name":"host","value":"ASCENDI-1GUJSNA"},{"name":"thread","value":"2564-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"browser_name","value":"'chrome'"},{"name":"data","value":"{'username': 'VaishnaviK123', 'password': 'Vaishu@123'}"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[]}],"tags":[]},"source":"97c51fbc1e4097b6.json","parameterValues":["'chrome'","{'username': 'VaishnaviK123', 'password': 'Vaishu@123'}"]}